ansible/templates/common/arg_vars.j2

```diff
@@ -3,8 +3,12 @@ For convenience in this walkthrough the main OMERO.web configuration options hav
 
 OMERO_USER={{ omero_user }}
 OMEROVER={{ omero_version }}
```

Comment:
still have 
``OMEROVER=OMERO-DEV-latest`` as the default
no needed when building the doc